### PR TITLE
NIP-57: Create the 'anon' tag

### PR DESCRIPTION
**Pull request by patrickReiis**
Since the NIP-57 defines event types to _recording_ lightning payments, it may be a good idea to add the feature of anonymous payments. I first saw this in [Snort](https://snort.social/).

### CODE REVIEW

**Comment by vitorpamplona** [comment]
+1. Amethyst also uses `anon`

File: README.md [hunk 1]
```diff
@@ -275,6 +275,7 @@ Please update these lists when proposing NIPs introducing new event kinds.
 | `title`           | article title                        | --                              | [23](23.md)                           |
 | `web`             | webpage URL                          | --                              | [34](34.md)                           |
 | `zap`             | pubkey (hex), relay URL              | weight                          | [57](57.md)                           |
+| `anon`            | empty string                         | --                              | [57](57.md)                           |
```
Comment (AsaiToshiya) [inline README.md:278]:
```suggestion
| `anon`            | --                                   | --                              | [57](57.md)                           |
```

Nit: Adjust the value to match the `encrypted` tag.

Comment (AsaiToshiya) [reply README.md:278]: I made a mistake.

**Comment by benthecarman** [comment]
Yeah I think #1064 would make more sense to merge, to at least document the functionality people are doing.

**Pull request closed without merging**